r/PACKAGING.md

```diff
@@ -18,86 +18,71 @@
-- [ ] Check [current CRAN check results](https://cran.rstudio.org/web/checks/check_results_arrow.html)
+- [ ] Check [current CRAN check results](https://cran.rstudio.org/web/checks/check_results_arrow.html).
 - [ ] Ensure the contents of the README are accurate and up to date.
 - [ ] Run `urlchecker::url_check()` on the R directory at the release candidate.
   commit. Ignore any errors with badges as they will be removed in the CRAN release branch.
-- [ ] [Polish NEWS](https://style.tidyverse.org/news.html#news-release) but do **not** update version numbers (this is done automatically later). You can find commits by, for example, `git log --oneline <sha of last release>..HEAD | grep "\[R\]"`
-- [ ] Run preliminary reverse dependency checks using `archery docker run r-revdepcheck`.
-- [ ] For major releases, prepare tweet thread highlighting new features.
+- [ ] [Polish NEWS](https://style.tidyverse.org/news.html#news-release) but do **not** update version numbers (this is done automatically later). You can find commits by, for example, `git log --oneline <sha of last release>..HEAD | grep "\[R\]"`.
+- [ ] For major releases, prepare content for social media highlighting new features.
 
-Wait for the release candidate to be cut:
+_Wait for the release candidate to be created._
 
-## After release candidate has been cut
+## After the Arrow Release Candidate Has Been Created
 
-- [ ] Create a CRAN-release branch from the release candidate commit, name the new branch e.g. `maint-X.X.X-r` and push to upstream
+- [ ] Create a CRAN-release branch from the release candidate commit, name the new branch e.g. `maint-X.X.X-r` and push to upstream.
 
-## Prepare and check the .tar.gz that will be released to CRAN.
+## Prepare and Check Package That Will Be Released to CRAN
 
-- [ ] `git fetch upstream && git checkout release-X.X.X-rcXX && git clean -f -d`
-- [ ] Run `make build`. This copies Arrow C++ into tools/cpp, prunes some
-  unnecessary components, and runs `R CMD build` to generate the source tarball.
-  Because this will install the package, you will need to ensure that the version
-  of Arrow C++ available to the configure script is the same as the version
-  that is vendored into the R package (e.g., you may need to unset `ARROW_HOME`).
-- [ ] `devtools::check_built("arrow_X.X.X.tar.gz")` locally
-- [ ] Run reverse dependency checks using `archery docker run r-revdepcheck`.
+- [ ] `git fetch upstream && git checkout maint-X.X.X-r && git clean -f -d`.
+- [ ] Run `make build`. This copies Arrow C++ into tools/cpp, prunes some unnecessary components, and runs `R CMD build` to generate the source tarball. Because this will install the package, you will need to ensure that the version of Arrow C++ available to the configure script is the same as the version that is vendored into the R package (e.g., you may need to unset `ARROW_HOME`).
```

Comment:
the long line is a bit harder  to read in source form but we are not keeping to any line length so :shrug: